multiqc_ngi/multiqc_ngi.py

```diff
@@ -236,8 +236,15 @@ def get_ngi_project_metadata(self, pid):
             infix = 'qc'
         else:
             infix = 'pipeline'
-        config.output_fn_name = f'{p_summary["project_name"]}_{infix}_{config.output_fn_name}'
-        config.data_dir_name = f'{p_summary['project_name']}_{config.data_dir_name}'
+        #If a filename is provided
+        if config.filename:
+            infix = f"{infix}_{config.filename}"
+            config.data_dir_name = f"{config.filename}_{config.data_dir_name}"
```

Comment:
shouldn't this be dir_name / filename instead of filename / dir_name?

Reply:
filename = our filename, and dir_name is the actual file name generated by multiqc. @aanil will double check.

Reply:
`p_summary['project_name']` was changed to `config.filename` in the MultiQC v22, and we used the same order in the previous version.